src/swc_util.rs

```diff
@@ -16,6 +16,10 @@ lazy_static! {
   static ref JS_DOC_RE: Regex = Regex::new(r#"\s*\* ?"#).unwrap();
 }
 
+pub(crate) fn is_false(b: &bool) -> bool {
+  !b
+}
```

Comment:
Oof.

Reply:
Is there a better way?

Just trying to strip out "useless" fields from the JSON serialization, as with large doc structures, it becomes a "problem".  `lib.dom.d.ts` for example is 9.9mb serialized. 😱

Reply:
I feel like we should use this and stuff like `skip_serializing_if = "Option::is_none"`  liberally one day.

Reply:
> Is there a better way?

No, I think this is right. Still pretty funny though 😅